src/box/tuple.h

```diff
@@ -331,6 +331,8 @@ struct PACKED tuple
 	 * be clarified by transaction engine.
 	 */
 	bool is_dirty : 1;
+        /** Flag indicates, that tuple contains compressed fields. */
+        bool is_compressed : 1;
```

Comment:
Do we really need this flag? Can we check the format instead?

Reply:
Yes fixed